src/components/header/DiscoverHeaderButton.js

```diff
@@ -50,10 +51,14 @@ export default function DiscoverHeaderButton() {
     navigate,
   ]);
 
-  const onLongPress = useCallback(
-    () => navigate(Routes.PROFILE_SHEET, { address: 'mikedemarais.eth' }),
-    [navigate]
-  );
+  const onLongPress = useCallback(() => {
+    navigate(Routes.PROFILE_SHEET, { address: 'mikedemarais.eth' });
+    analytics.track('Viewed ENS profile', {
```

Comment:
seems like we only use this method in __DEV__ so we shouldn't need it
![image](https://user-images.githubusercontent.com/12115171/160926471-06a38a33-5ffb-4324-a1d7-acbbb8553a36.png)